packages/common/index.ts

```diff
@@ -679,27 +679,30 @@ export class GetAlertsResponse {
   alerts!: Alert[];
 }
 
-export class SubmitCourseParams {
-  @IsString()
-  coordinator_email!: string;
-
+/**
+ * Represents the parameters for a course being registered for register_courses endpoint.
+ * @param name - The name of the section group.
+ * @param displayName - user friendly display name entered by Prof
+ * @param semester - The name of the semester.
+ * @param iCalURL - The URL for the iCal calendar.
+ * @param coordinator_email - The email for the course coordinator.
+ * @param timezone - The timezone derived from the Campus field on the form.
+ */
+export class RegisterCourseParams {
   @IsString()
   name!: string;
```

Comment:
Can we actually change the names of these Params to "sectionGroupName" and "name" instead of "displayName" since that's what we use on the backend